frontend/packages/topology/src/utils/createAggregateEdges.ts

```diff
@@ -55,13 +81,14 @@ const createAggregateEdges = (
               source,
               target,
               id: `${source}_${target}`,
```

Comment:
Just to be safe, could we prefix this id:
```suggestion
              id: `aggregate:${source}_${target}`,
```